src/sentry/models/grouphistory.py

```diff
@@ -24,13 +24,16 @@
 class GroupHistoryStatus:
     # Note that we don't record the initial group creation unresolved here to save on creating a row
     # for every group.
+
+    # UNRESOLVED should be used for ONGOING groups.
+    # Use REGRESSED/ESCALATING to specify other substatuses.
     UNRESOLVED = 0
     RESOLVED = 1
     SET_RESOLVED_IN_RELEASE = 11
     SET_RESOLVED_IN_COMMIT = 12
     SET_RESOLVED_IN_PULL_REQUEST = 13
     AUTO_RESOLVED = 2
-    IGNORED = 3
+    IGNORED = 3  # use the ARCHIVED_XXX statuses instead
```

Comment:
I don't think we can assume the default of `ARCHIVED_FOREVER` for GroupHistory -- we don't really distinguish between the nuances of the ignored status. We can backfill the Group.status column because we'd have enough information about the group's current status. 

If we deprecate a status from GroupHistory, we can probably just leave the existing rows since it captures the accurate history from the time the change was made.